src/utils/githubEventHelpers.ts

```diff
@@ -192,6 +192,31 @@ export async function getKeyValueFromProjectField(
   return response?.node.fieldValueByName?.name;
 }
 
+export async function getIssueDueDateFromProject(
+  issueNodeId: string,
+  octokit: Octokit
+) {
+  const query = `query{
+    node(id: "${issueNodeId}") {
+        ... on ProjectV2Item {
+          id
+          fieldValueByName(name: "Respond By") {
+            ... on ProjectV2ItemFieldTextValue {
+              text
+            }
+          }
+        }
+      }
+    }`;
+
+  const data = {
+    issueNodeId,
+  };
+  const response = await sendQuery(query, data, octokit);
+
+  return response?.node.fieldValueByName?.text;
+}
```

Comment:
Lotta uncovered lines here. 🐭

Reply:
You got me 😅, it seems like writing tests here are not super useful, but I guess minimal tests are better than no tests. Will add.

Reply:
> it seems like writing tests here are not super useful

Maybe not, I was just going off of the noise Codecov was making. 😅